backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/SysMLElementSerializer.java

```diff
@@ -775,6 +788,47 @@ private void appendArgumentMember(Appender builder, ParameterMembership paramete
                     .forEach(val -> this.appendOwnedExpression(builder, val.getValue()));
         }
     }
+    
+    private void appendArgumentList(Appender builder, InvocationExpression expression) {
+        builder.append(LabelConstants.OPEN_PARENTHESIS);
+        Relationship relationship = expression.getOwnedRelationship().stream()
+                   .filter(FeatureMembership.class::isInstance)
+                   .map(FeatureMembership.class::cast)
+                   .findFirst()
+                   .orElse(null);
+        if (relationship instanceof ParameterMembership) {
+            appendPositionalArgumentList(builder, expression);
+        }
+        else if (relationship instanceof FeatureMembership) {
+            LOGGER.warn("NamedArgumentList are not handled yet");
+        }
+        builder.append(LabelConstants.CLOSE_PARENTHESIS);
+    }
+    
+    private void appendPositionalArgumentList(Appender builder, InvocationExpression expression) {
+        List<ParameterMembership> parameterMemberships = expression.getOwnedRelationship().stream()
+            .filter(ParameterMembership.class::isInstance)
+            .map(ParameterMembership.class::cast)
+            .filter(param -> {
+                if (param.getOwnedMemberParameter() != null) {
+                    return param.getOwnedMemberParameter().getOwnedRelationship().stream()
+                    .filter(FeatureValue.class::isInstance)
+                    .map(FeatureValue.class::cast)
+                    .findAny().isPresent();
```

Comment:
`isPresent()` should be on the next line

Reply:
done